README.md

```diff
@@ -91,13 +184,108 @@ You then need to do a second call to the `signIn` mutation providing only the `o
 }
 ```
 
-## OAuth
+## OAuth / Login with Sorare
+
+If you want to make requests on behalf of other users you can rely on the [OAuth](https://oauth.net/2/) protocol.
+
+Please reach out to us first through our [Help Center](https://help.sorare.com/hc/en-us/requests/new) and ask us to generate you some OAuth credentials.
+
+We need:
+
+- the name of your App
+- the callback URIs of your OAuth applications
+  - ex: `http://localhost:3000/auth/sorare/callback` (for dev) & `https://myapp.com/auth/sorare/callback` (for prod)
+- the PNG version of your App logo
+
+### OAuth Credentials
+
+Once manually validated, we'll provide you with:
```

Comment:
Once we validate your application, you will be provided with:
- OAuth Client ID
- OAuth Secret